docker-compose.yml

```diff
@@ -22,6 +22,8 @@ services:
     restart: unless-stopped
     image: mysql:5.7
     container_name: cbioportal-database-container
+    ports:
+      - "3306:3306"
```

Comment:
for production use often one prolly wouldn't want to expose 3306 because it opens up the port to any process on the machine. This is protected atm by running this cbioportal-database container within `cbio-net` network (see `networks` section). That is, only containers on this network are able to access 3306. To connect to the db with the mysql command you can either use `exec` to run a mysql command within the cbioportal-database container. Or you can start a separate mysql container on the `cbio-net`. Not sure if those alternatives cover your use case?

Reply:
I would propose to add a separate -dev compose file that may also do the follwoing:

- Enable Debug for init script (`SHOW_DEBUG_INFO: "true"`)
- Expose MySQL port
- Expose session service port
- Expose MongoDB port for session service

Also I may add a PR that easily allows so set a different port for cBioPortal as 8080 is very common in may already be in use. Of course it's possible for everyone to change the compose file, but that might cause unnecessary merge conflicts.

Reply:
This can be best done with a compose extension file like so:

_db-ports.yml_
```
services:
   cbioportal-database-container:
   ports:
      - "3306:3306"
```

And then run:
```
docker-compose -f docker-compose.yml -f db-ports.yml up -d
```